src/libs/actions/Policy/Policy.ts

```diff
@@ -3454,6 +3454,60 @@ function upgradeToCorporate(policyID: string, featureName: string) {
     API.write(WRITE_COMMANDS.UPGRADE_TO_CORPORATE, parameters, {optimisticData, successData, failureData});
 }
 
+function setWorkspaceDefaultSpendCategory(policyID: string, groupID: string, category: string) {
```

Comment:
It optimistically sets a new category for the given mcc (merchant category code). In case it fails it clears the changes made, otherwise it follows thorough with the response.